packages/block-library/src/navigation/index.php

```diff
@@ -371,24 +254,10 @@ function block_core_navigation_maybe_use_classic_menu_fallback() {
  */
 function block_core_navigation_get_most_recently_published_navigation() {
 
-	// Default to the most recently created menu.
-	$parsed_args = array(
-		'post_type'              => 'wp_navigation',
-		'no_found_rows'          => true,
-		'update_post_meta_cache' => false,
-		'update_post_term_cache' => false,
-		'order'                  => 'DESC',
-		'orderby'                => 'date',
-		'post_status'            => 'publish',
-		'posts_per_page'         => 1, // get only the most recent.
-	);
-
-	$navigation_post = new WP_Query( $parsed_args );
-	if ( count( $navigation_post->posts ) > 0 ) {
-		return $navigation_post->posts[0];
-	}
+	// add notice about deprecation
+	_deprecated_function( __FUNCTION__, 'X.X.X', 'WP_Navigation_Gutenberg::get_most_recently_published_navigation' );
 
-	return null;
+	return WP_Navigation_Gutenberg::get_most_recently_published_navigation();
```

Comment:
We should leave all the functions that we deprecate intact so what people who use them don't get surprising results when we chanage things in `WP_Navigation_Gutenberg` in the future.